src/org/opendatakit/briefcase/ui/automation/AutomationPanelForm.java

```diff
@@ -0,0 +1,164 @@
+package org.opendatakit.briefcase.ui.automation;
+
+import static org.opendatakit.briefcase.ui.reused.FileChooser.directory;
+
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+import java.util.function.Consumer;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+import org.opendatakit.briefcase.automation.AutomationConfiguration;
+import org.opendatakit.briefcase.export.ExportConfiguration;
+import org.opendatakit.briefcase.model.BriefcasePreferences;
+import org.opendatakit.briefcase.ui.export.components.ConfigurationDialog;
+import org.opendatakit.briefcase.util.StringUtils;
+
+@SuppressWarnings("checkstyle:MethodName")
+public class AutomationPanelForm {
+  public JPanel container;
+  private JLabel scriptDirLabel;
+  private JTextField scriptDirField;
+  private JButton scriptDirChooseButton;
+  private JButton generateScriptButton;
+  private JButton changeDefaultConfigurationButton;
```

Comment:
Since we're also going to have the pull and push configurations, I think it would be better to rename this to `setExportConfigurationButton`